src/api/app/components/bs_request_comment_component.html.haml

```diff
@@ -24,12 +24,17 @@
                                     action: comment_path(comment) },
                             class: 'dropdown-item delete_link', title: 'Delete comment') do
                 Delete
-      .px-3.py-2.border-bottom
-        - unless @nodecoration
-          = link_to(helpers.realname_with_login(comment.user), user_path(comment.user))
-          \-
-        = link_to_comment(comment)
-        = render CommentHistoryComponent.new(comment)
+      .px-3.py-2.border-bottom.d-flex.flex-column.flex-lg-row.justify-content-between.align-items-start
+        .d-inline-flex.justify-content-start
+          - unless @nodecoration
+            = link_to(helpers.realname_with_login(comment.user), user_path(comment.user))
+            \-
+          = link_to_comment(comment)
+          = render CommentHistoryComponent.new(comment)
+        .d-inline-flex.justify-content-end
```

Comment:
Adding a `small` class here will make the badges a bit smaller.

Reply:
Done